### ISSUE TITLE
Is there a simple example for creating text messages?

### ISSUE THREAD

**Issue by apwelsh** [badoo/Chatto#492]
I am trying to create a new application based on my library.  The demo app is very convoluted and abstract.  Are there any simple concrete examples on creating a messaging app for simple text messages?  No need for avatars, or any other frills.  Just a simple text messaging app.  tying to understand this library at the most basic level.

**Comment by czater** [comment]
There is a tutorial on Udemy that explain everything very good ;)

**Comment by apwelsh** [comment]
Which one?  A search for chatto turns up nothing.

**Comment by czater** [comment]
Build a Powerful iOS Chat App in Swift 3

**Comment by apwelsh** [comment]
Yeah, that is what I was afraid of.  That course has been taken down, and only available to people that purchased it.  I found that one earlier through a google search.
